pkg/apis/options/legacy_options.go

```diff
@@ -488,6 +488,8 @@ type LegacyProvider struct {
 	KeycloakGroups                         []string `flag:"keycloak-group" cfg:"keycloak_groups"`
 	AzureTenant                            string   `flag:"azure-tenant" cfg:"azure_tenant"`
 	AzureGraphGroupField                   string   `flag:"azure-graph-group-field" cfg:"azure_graph_group_field"`
+	AzureOIDCSkipGroupsFromGraph           bool     `flag:"azure-oidc-skip-groups-from-graph" cfg:"azure_oidc_skip_groups_from_graph"`
+	AzureOIDCMutliTenantAllowedTenants     []string `flag:"azure-oidc-multi-tenant-allowed-tenant" cfg:"azure_oidc_multi_tenant_allowed_tenants"`
```

Comment:
This name seems a bit convoluted. Wouldn't `AzureAllowedTenants` be enough?

Reply:
Right, this variable is too long, I wanted to stick this with the flag name. Flag name is also a bit convoluted, but this setting applies only to the multi-tenant app registrations and I wanted to highlight it.